cloudquery/sdk/serve/plugin.py

```diff
@@ -133,13 +159,88 @@ def run(self, args):
             help="network to serve on. can be tcp or unix",
         )
 
-        parsed_args = parser.parse_args(args)
+    def _register_package_command(self, subparsers):
+        package_parser = subparsers.add_parser("package", help="Package plugin for publishing")
+        package_parser.add_argument(
+            "--log-format",
+            type=str,
+            default="text",
+            choices=["text", "json"],
+            help="logging format",
+        )
+        package_parser.add_argument(
+            "--log-level",
+            type=str,
+            default="info",
+            choices=["trace", "debug", "info", "warn", "error"],
+            help="log level",
+        )
+        package_parser.add_argument(
+            "--dist-dir",
+            type=str,
+            default="dist",
+        )
+        package_parser.add_argument(
+            "--message",
+            type=str,
+            default="",
+        )
 
-        if parsed_args.command == "serve":
-            self._serve(parsed_args)
-        else:
-            parser.print_help()
-            sys.exit(1)
+    def _package(self, args):
+        def run_docker_cmd(cmd):
+            result = subprocess.run(cmd,capture_output=True)
+            if result.returncode != 0:
+                err = "" if result.stderr is None else result.stderr.decode('ascii').strip()
+                raise ChildProcessError("Unable to run Docker command: %s" % err)
+
+        tmp_dist_dir = "%s/plugin" % args.dist_dir
+        Path(tmp_dist_dir).mkdir(0o755, exist_ok=True, parents=True)
+
+        package_json = self._make_package_json(args.message)
+
+        for i, dockerfile in enumerate(self._plugin.options().dockerfiles):
+            image_name = "cq-docker-image-%d" % i
```

Comment:
I would keep the naming convention we have for zip files, see https://github.com/cloudquery/plugin-sdk/blob/0d43ffd77b0d96ef4b0e1b9e7ad9a616d9b6211c/serve/package.go#L111